commands/git/git.md

```diff
@@ -0,0 +1,7 @@
+# Git :octocat:
```

Comment:
Do we have any other option for the icon here? This icon does not matches with the text size.